README.md

```diff
@@ -75,4 +79,7 @@ Figaro allows you to easily configure the environment variables from `config/app
 
 Worth noting that auth.berkeley.edu authentication is currently registered for the domain “https://communities-cal.herokuapp.com". Since currently it is tied to an old team member's account, you may need to re-request authentication to change this.
 
+### Heroku Database Commands
+To reset the database on heroku: `heroku restart && heroku pg:reset DATABASE_URL --confirm communities-cal && heroku run rake db:migrate && heroku run rake db:seed`
```

Comment:
Looks good! This is based off [this gist](https://gist.github.com/zulhfreelancer/ea140d8ef9292fa9165e)!